apps/charterafrica/src/pages/api/v1/resources/ecosystem/proxy.page.js

```diff
@@ -0,0 +1,23 @@
+import fetchJson from "@/charterafrica/utils/fetchJson";
```

Comment:
We can just have a schema action rather than this proxy. Our actions would then be

1. `list` if we use **PUT** method or `update-list` if we want to stick to **GET**.
2. `content` / `update-content`
3. `schema` (GET only. Again, this going to external service I feel like it should be protected but not sure how to pass the secret key to Payload without exposing it publicly)

PS: I think we can still use airtable.js to fetch bases and tables, no?

Reply:
Looks like Airtable.js doesn't have an API to list all bases unless we use REST API

Reply:
😱 ... that's just sad!